File: Day-6/p1.c
Problem statement :

c program to generate first n Triangular numbers

solution :

include <stdio.h>
void triangular_series(int n)
{

for (int i = 1; i <= n; i++)  

    printf(" %d ", i*(i+1)/2); 
}

int main()
{

int n ;
printf("Enter value for n\n");

scanf("%d",&n);

triangular_series(n); 

return 0;
}

output :

Enter value of n
1 3 6 10 15 21 28 36 45 55

File: Day-11/p1.py
problem statement :

Python program to read a number and print the pattern

solution :

rows = int(input("Enter the number of rows "))
for i in range(rows + 1, 0, -1):
    for j in range(0, i - 1):
        print("*", end=' ')
    print(" ")

output :

Enter the number
5
* * * * * 

* * * * 

* * * 

* * 

*
